.circle/install-mongo.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+sudo apt-get purge mongodb-org*
```

Comment:
And yes, to clarify - ideally we would always be able to use native service functionality without needing to hack stuff on top ourselves, but sadly this is usually only possible for very simple and basic use cases.

If you want better performance and / or control version, sadly this won't work anymore.